src/pytti/workhorse.py

```diff
@@ -9,6 +9,7 @@
 from pathlib import Path
 import os
 import re
+from sqlite3 import NotSupportedError
```

Comment:
the fuck? this must've been codex getting cheeky when I wasn't paying attention.